test/testContracts/ForeignBridgeV2.sol

```diff
@@ -1,17 +1,22 @@
 pragma solidity ^0.4.19;
 
 
-import "../../contracts/upgradeable_contracts/U_ForeignBridge.sol";
+import "../../contracts/upgradeable_contracts/native_to_erc20/ForeignBridgeNativeToErc.sol";
 
 
 interface OwnableToken {
     function transferOwnership(address) external;
 }
 
-contract ForeignBridgeV2 is ForeignBridge {
+contract ForeignBridgeV2 is ForeignBridgeNativeToErc {
     function changeTokenOwnership(address _newTokenOwner) public onlyOwner {
         address token = address(erc677token());
         OwnableToken poa = OwnableToken(token);
         poa.transferOwnership(_newTokenOwner);
     }
+
+    address public something;
```

Comment:
Why is it needed?

Reply:
for tests

Reply:
so could you add a comment to make it more obvious?